pkg/controller/constraint/constraint_controller.go

```diff
@@ -461,12 +413,7 @@ func (r *ReconcileConstraint) Reconcile(ctx context.Context, request reconcile.R
 					enforcementAction: enforcementAction,
 					status:            metrics.ErrorStatus,
 				})
-				status.Status.Errors = append(status.Status.Errors, constraintstatusv1beta1.Error{Message: err.Error()})
-				if err2 := r.writer.Update(ctx, status); err2 != nil {
-					log.Error(err2, "could not report constraint error status")
-				}
-				reportMetrics = true
```

Comment:
What happens to the `reportMetrics` flag?

Reply:
Added the flag back. This was not supposed to be removed. Good catch!